graph_nets/blocks.py

```diff
@@ -446,6 +446,7 @@ def _build(self, graph):
         `use_receiver_nodes` or `use_sender_nodes` is `True`) and per graph
         globals (if `use_globals` is `True`) should be concatenable on the last
         axis.
+      **kwargs: Optional keyword arguments to pass to the Sonnet module.
```

Comment:
"to pass to the `edge_model`", since technically it may not be a sonnet module.

(Here and in other blocks)